onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -622,6 +639,47 @@ def _should_skip_test_sample(op_name: str, sample) -> Optional[str]:
     return None
 
 
+class TestFunctionValidity(unittest.TestCase):
+    def test_all_script_functions_are_onnx_functions(self):
+        for func_with_wrangler in OPINFO_FUNCTION_MAPPING_SCRIPTED.values():
+            if isinstance(func_with_wrangler, tuple):
+                func = func_with_wrangler[0]
+            else:
+                func = func_with_wrangler
+            if not isinstance(func, onnxscript.OnnxFunction):
+                raise AssertionError(
+                    f"'{func}' is not an OnnxFunction. Was it decorated with '@torch_op'?"
+                    "If the function is trace_only, please move it to the "
+                    "'OPINFO_FUNCTION_MAPPING_TRACE_ONLY' dict."
+                )
+
+    def test_all_trace_only_functions_are_not_onnx_functions(self):
+        for func_with_wrangler in OPINFO_FUNCTION_MAPPING_TRACE_ONLY.values():
+            if isinstance(func_with_wrangler, tuple):
+                func = func_with_wrangler[0]
+            else:
+                func = func_with_wrangler
+            if isinstance(func, onnxscript.OnnxFunction):
+                raise AssertionError(
+                    f"'{func.name}' is an OnnxFunction. "
+                    "If the function is not trace_only, please move it to the "
+                    "'OPINFO_FUNCTION_MAPPING_SCRIPTED' dict."
+                )
+
+    @parameterized.parameterized.expand(list(OPINFO_FUNCTION_MAPPING_SCRIPTED.items()))
+    @unittest.skipIf(
+        onnx_older_than("1.14"),
+        "Function checker is not available before ONNX 1.14",
+    )
+    def test_script_function_passes_checker(self, _, func_with_wrangler):
+        if isinstance(func_with_wrangler, tuple):
+            func = func_with_wrangler[0]
+        else:
+            func = func_with_wrangler
+        function_proto = func.to_function_proto()
+        onnx.checker.check_function(function_proto)
```

Comment:
Do you know why mypy complains `  Error (MYPY) attr-defined
    Module has no attribute "check_function"`? Is it because lint CI is not using onnx-weekly package?

Reply:
hmm yes